platform/src/common/components/Charts/components/index.jsx

```
@@ -19,122 +22,87 @@ export const truncate = (str) => {
+      AirQualityIcon: icon,
+      airQualityColor: color,
+    };
+  }
+
+  // Ensure ranges are sorted descendingly
+  const sortedRanges = [...ranges].sort((a, b) => b.limit - a.limit);
+
+  // Iterate through the sorted ranges to find the correct category
+  for (let i = 0; i < sortedRanges.length; i++) {
+    if (value > sortedRanges[i].limit) {
+      continue; // Skip if value exceeds the current limit
```

Biome

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)